runtime/src/ancient_append_vecs.rs

```diff
@@ -3,13 +3,14 @@
 //! 1. a slot that is older than an epoch old
 //! 2. multiple 'slots' squashed into a single older (ie. ancient) slot for convenience and performance
 //! Otherwise, an ancient append vec is the same as any other append vec
-#![allow(dead_code)]
 use {
     crate::{
-        accounts_db::FoundStoredAccount,
+        accounts_db::{AccountStorageEntry, AccountsDb, FoundStoredAccount},
+        accounts_index::AccountIndexGetResult,
```

Comment:
I am guessing either `checks` or `clippy` will yell about these unused `use` items. Looks like this whole file can be reverted.

Reply:
yep, fixed